base/src/commonMain/kotlin/utils/UnitSystem.kt

```diff
@@ -22,12 +22,14 @@ enum class UnitSystem {
     METRIC, MIXED, IMPERIAL;
 
     companion object {
+        // FIXME IMPERIAL and US customary seem reversed
         fun withRawValue(value: String): UnitSystem = when (value) {
             "U.S." -> IMPERIAL
             "U.K." -> MIXED
             else -> METRIC
```

Comment:
What (ISO) format is "U.S." and "U.K."? Based on https://github.com/splendo/kaluga/pull/394/files/908a6902e1d86e80e1a68015e038d6d289c6c3e9#r720262973 I would use "US_CUSTOMARY", "UK_CUSTOMARY", ...

Reply:
PS: I would still use the METRIC system as default and not SI

Reply:
Not my code but I see this is used for Locale. Probably will have to merge with @vpodlesnyak approach. Will discuss with him.